packages/botonic-plugin-google-translate/src/translator.ts

```diff
@@ -9,8 +9,9 @@ export class Translator {
     const responses = await Promise.all(
       targets.map(t => this.service.translateText(text, t))
     )
-    const translations = {}
-    responses.forEach((r, i) => (translations[targets[i]] = r.translatedText))
-    return translations
+    return Object.assign(
```

Comment:
I hadn't realized you returned an object instead of a list,otherwise your initial solution was ok.
But your new solution is also cool :+1: